testdata/config/prometheus.yaml

```diff
@@ -34,16 +34,24 @@ spec:
         numFiniteBuckets: 8
         scale: 1
         growthFactor: 10
+  - name: tcp_bytes_sent
+    instance_name: bytessent.metric.istio-config-default
+    kind: DISTRIBUTION
+    label_names: [ source, target, service, method, version, response_code ]
```

Comment:
if possible, i'd prefer to use the metrics definitions in my outstanding PR.

In particular, things like `method` and `response_code` are not applicable, and I'd like to clarify the names around `source`, `target`, and which one `version` applies to.  Also, I think this should be a counter (and not a distribution), as this will like accumulate through the course of a connection (and counter would enable easy bytes/second charts, etc.)

Reply:
I am happy to exclude these files from the PR, if your PR already does it in a different way.

Reply:
cool. even if they go in, i can rebase and reshape them in my TCP PR.